.buildkite/release-automation/pre_release.rayci.yml

```diff
@@ -39,3 +40,30 @@ steps:
       env:
         AUTOMATIC: 1
         RELEASE_FREQUENCY: "nightly"
+
+  - block: "Trigger Release nightly-3x test"
+    depends_on: check-commit-hash
+
+  - label: "Trigger Release nightly-3x test"
```

Comment:
add a `depends_on` on the block step for these two trigger steps?

otherwise the `weekly` test will `depend_on` all block steps before it.

Reply:
done!